### PR TITLE
Enterprise Docs references to external blogs as a source of core information (Part 1)

### PR DESCRIPTION
**Pull request by yesamer**
In this PR, I moved the Blog related to the Test Scenario only.
The Import Java Class we be managed in another PR.

Kudos to @kbowers-ibm who provided the first version of the documentation.

### CODE REVIEW

**Comment by LightGuard** [comment]
This looks good, pending another review, I'm happy to merge.

File: tools/developer-tools-for-vscode.adoc [hunk 1]
```diff
@@ -1,17 +1,117 @@
...
+Static file validation, DMN Runner, and Dev deployments are not available for {DEV_TOOLS} yet, but you can achieve a similar experience of DMN Runner with our Test Scenario Editor.
 See how below.
 
-== Running unit tests with Scenario Simulation files
+== Running unit tests with Test Scenario
 
-Running Scenario Simulation (.scesim) files requires a configured Java project with the JUnit dependency available.
-Please follow this https://blog.kie.org/2021/04/how-to-use-test-scenario-editor-to-test-your-dmn-asset-in-vscode.html[blog post tutorial] to get started with Scenario Simulation on {PRODUCT_SHORT} Developer Tools.
+=== Overview
+This documentation provides a step-by-step guide on how to use the Test Scenario editor in VSCode to test DMN assets created with Kogito Tooling. For that scope, this guide will refer to an existing project that lives in the https://github.com/apache/incubator-kie-kogito-examples/tree/stable/kogito-quarkus-examples/dmn-pmml-quarkus-example[kie-kogito-examples] GitHub repo. However, this guide will provide the steps to enable Test Scenario in your own project
```
Comment (kbowers-ibm) [inline tools/developer-tools-for-vscode.adoc:14]: Is Kogito Tooling okay here, or should we have something IBM related, e.g. "created with IBM BAMOE..."

File: tools/developer-tools-for-vscode.adoc [hunk 2]
```diff
@@ -1,17 +1,117 @@
...
+3. **Configure Test Scenario**:
+   The editor will populate columns based on the DMN asset. `GIVEN` columns are for inputs and `EXPECT` columns are for outputs.
++
+image::developer-tools/scesim-empty-file.png[]
+4. **Define Test Cases**:
+   Fill in the test data. For example:
++
+[source,console]
+----
+   GIVEN fdl1 = 3, fdl2 = 2, fdl3 = "y"
```
Comment (kbowers-ibm) [inline tools/developer-tools-for-vscode.adoc:84]:
```suggestion
   GIVEN fld1 = 3, fld2 = 2, fld3 = "y"
```

File: tools/developer-tools-for-vscode.adoc [hunk 3]
```diff
@@ -1,17 +1,117 @@
...
+
+1. Clone the `incubator-kie-kogito-examples` repository.
+2. Open VSCode and navigate to `incubator-kie-kogito-examples/kogito-quarkus-examples/dmn-pmml-quarkus-example`.
+This project contains all the necessary files and prerequisites for this tutorial.
+
+==== Setup any project
+To enable Test Scenario in your project, follow the below steps
+
+1. **Add the required Test Scenario dependency**:
+   Add the following dependency to your projects's `pom.xml` to enable the Test Scenario runner:
```
Comment (kbowers-ibm) [inline tools/developer-tools-for-vscode.adoc:38]:
```suggestion
   Add the following dependency to your project's `pom.xml` to enable the Test Scenario runner:
```

File: tools/developer-tools-for-vscode.adoc [hunk 4]
```diff
@@ -1,17 +1,117 @@
...
+=== Project Setup
+==== Cloning the Example Project
+If you are a new {PRODUCT_SHORT} user and you don't have any project to set up yet, or you prefer having an out-of-the-box project to deal with, you can use the previously mentioned example in the `incubator-kie-kogito-examples` repository.
+
+1. Clone the `incubator-kie-kogito-examples` repository.
+2. Open VSCode and navigate to `incubator-kie-kogito-examples/kogito-quarkus-examples/dmn-pmml-quarkus-example`.
+This project contains all the necessary files and prerequisites for this tutorial.
+
+==== Setup any project
+To enable Test Scenario in your project, follow the below steps
```
Comment (kbowers-ibm) [inline tools/developer-tools-for-vscode.adoc:35]:
```suggestion
To enable Test Scenario in your project, follow the below steps:
```

**Review by kbowers-ibm — APPROVED** [review]
Thanks @yesamer, LGTM! Just a few minor suggestions inline:

File: tools/developer-tools-for-vscode.adoc (hunk 1 above)
Comment (yesamer) [reply tools/developer-tools-for-vscode.adoc:14]: @kbowers-ibm Thank you, change applied.

**Pull request merged**